paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerAdapter.kt

```diff
@@ -51,6 +55,29 @@ interface CustomerAdapter {
      * setupIntentClientSecretProvider in the init call.
      */
     suspend fun setupIntentClientSecretForCustomerAttach(): Result<String>
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    companion object {
+
+        /**
+         * Creates an instance of [StripeCustomerAdapter]
```

Comment:
Maybe something along the lines of creates a default implementation of `CustomerAdapter` which uses the Stripe customer object as the data store? (please modify that! LOL)